docs/features/telemetry-serilog.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Home"
+layout: default
+---
+
+# Telemetry (Serilog)
+
+## Version Enricher
+
+The `Arcus.Observability.Telemetry.Serilog` library provides a [Serilog enricher](https://github.com/serilog/serilog/wiki/Enrichment) 
```

Comment:
We should talk about what it adds exactly, in this case `version` property with `1.0.0.0-preview` or so